gulpfile.js

```diff
@@ -324,4 +324,12 @@ gulp.task('dev:runserver', gulp.series(function(done) {
     done();
```

Comment:
@live-wire Please remove this file from here and update the PR description. I have opened a PR #1771 for the gulp staging fix.